packages/decorators/package.json

```diff
@@ -27,6 +27,7 @@
 	"devDependencies": {
 		"@sapphire/discord-utilities": "workspace:^",
 		"@sapphire/discord.js-utilities": "workspace:^",
+		"@sapphire/plugin-subcommands": "pr-271",
```

Comment:
I'm fine with next but not with PR versions so I'm going to make this PR a draft until subcommand is merged

Reply:
Drafted. Feel free to mark as resolved when read.